ol3_viewer_prepare.sh

```diff
@@ -1,6 +1,23 @@
 #!/bin/bash
 
-echo "preparing ol3-viewer"
-rm  -rf libs
-mkdir -p libs
-ant
+compile_viewer=0
+if [ "$#" -gt 0 ] && [ "$1" = "DEV" ]; then
+    if [ -f "libs/ol3-viewer.js" ]; then
+        timestamp=$(date +%s)
+        echo $timestamp
+        timestamp_viewer=$(stat -f%c libs/ol3-viewer.js)
```

Comment:
also I think that the echo $timestamp slipped in there for debug info...

is it a problem to stick to upper case variables for bash scripts. it seems to be kind of a style guide for shell scripts (as far as I have experienced it)